evaluation/run_ycsb.py

```diff
@@ -9,6 +9,28 @@
 from pathlib import Path
 
 DBS = ["capybarakv", "pmemrocksdb", "redis"]
+experiment_config_file = None
+
+def is_windows():
+    return sys.platform == "win32"
+
+def get_ycsb_path():
+    if is_windows():
+        return ".\\YCSB\\bin\\ycsb.bat"
+    else:
+        return "./YCSB/bin/ycsb"
+
+def get_ycsb_config_path():
+    if is_windows():
+        return "capybarakv_config_win.toml"
+    else:
+        return "capybarakv_config.toml"
+
+def get_experiment_config_path():
+    if is_windows():
+        return "experiment_config_win.toml"
+    else:
+        return experiment_config_file
```

Comment:
I tried running this script on Linux and ran into an issue with this function returning `None` (I think because `experiment_config_file` is not marked `global`). Do you have a Linux/WSL instance you could test it on as well?